### PR TITLE
Remove hardcoded inlined display style for anchor editor

### PR DESCRIPTION
**Pull request by PSyton**
For better forms customization with CSS it's better to handle show|hide
state by class.
I've appended additional class for visible form and remove inline style
modifications.
Also I't moved show/hide form functional to form implementation.

### CODE REVIEW

**Comment by orthes** [comment]
Thanks! This makes sense. But can you remove `dist/js/medium-editor.js` from the pull request? Also I see that you removed an old test from `spec/anchor.spec.js`, but you didn't add a new one. Can you do that? Thank you.

**Comment by orthes** [comment]
Would you mind adding some tests as well?

**Comment by PSyton** [comment]
I will add tests this days

File: src/sass/components/_toolbar-form.scss [hunk 1]
```diff
@@ -38,3 +38,7 @@
         text-decoration: none;
     }
 }
+
+.medium-editor-toolbar-form-active {
+    display: block;
+}
```
Comment (j0k3r) [inline src/sass/components/_toolbar-form.scss]: Could you please add an empty line at the end of the file?

**Comment by PSyton** [comment]
_toolbar-form.scss already has empty line at the end.

**Comment by PSyton** [comment]
Added simple test

**Comment by orthes** [comment]
Great! Looks good. Thank you.

**Pull request merged**